packages/blade/src/components/Amount/_decisions/decisions.md

```diff
@@ -0,0 +1,75 @@
+| ------------- | ---------------------------------------------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| value         | `number`                                                   | `undefined` | The value to be rendered within the component                                                                                                                                                     | ✅       |
+| variant       | `positive`, `negative`, `notice`, `information`, `blue`    | `undefined` | The variant of the amount to be rendered                                                                                                                                                          |          |
+| fontWeight    | `bold`, `regular`                                          | `bold`      | The weight of the amount to be rendered                                                                                                                                                           |          |
+| suffix        | `none`, `decimals` , `humanize`                            | `decimals`  | The type of affix to be displayed with the currency value, (none, decimals, and humanize) that respectively indicate no affix, the number of decimal places, and the scale of the currency value. |          |
+| size          | `3xlarge`, `2xlarge`, `xlarge`, `large`, `medium`, `small` | `3xlarge`   | The size of the amount to be rendered                                                                                                                                                             |          |
+| isAffixSubtle | `true`, `false`                                            | `true`      | Highlight the main amount by making the prefix symbol and decimal digits small and faded                                                                                                          |          |
+
+### Sample Usage
+
+```jsx
+import { Amount } from '@razorpay/components';
+
+<Amount
+  suffix="decimals"
+  size="2xlarge"
+  isAffixSubtle={true}
+  fontWeight="regular"
+  variant="information"
+  value={1000}
+/>;
+```
+
+## Open Questions
+
+- Should we call `isSuffixPrefixHighlighted` or `isHighlighted` or `isAffixHighligted`, considering when this value is true the value will be highligted and not the affix.
+
+#### Dev
+
+- Should we call it `variant` or `intent`, `isStyled` or something more apt?
+- Should we separate the amount with commas based on the Indian currency format?
+- What should be the default `intent`? Kept it as `default` for now, but `neutral` seems more apt.
+- We currently use amount value in paise in backend, so how should we convert it to rupee here?
```

Comment:
How are these values generally sent? I thought it might be in rupees for example `0.01` INR?

Reply:
Currently in X, they use it in rupee and the value transformation is handled by the consumer.